Modules/Quotas/src/Quotas.Application/Metrics/Queries/ListMetrics/Handler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using AutoMapper;
+using Backbone.Modules.Quotas.Application.DTOs;
+using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using MediatR;
+
+namespace Backbone.Modules.Quotas.Application.Metrics.Queries.ListMetrics;
+public class Handler : IRequestHandler<ListMetricsQuery, IEnumerable<MetricDTO>>
```

Comment:
Create a response class, in order to stay consistent with other handlers like ListRelationships, ListMessages, etc.

Btw, I just noticed that you did the same mistake for the ListClients handlers. Can you fix that as well please?